extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1080-lms-assignment.sql

```diff
@@ -23,6 +23,7 @@ BEGIN
 		edfiSection.LocalCourseCode [LocalCourseCode],
 		edfiSection.[SessionName] [SessionName],
 		edfiSection.[SchoolYear] [SchoolYear],
+		edfiSection.SchoolId,
```

Comment:
Is there anything you can update in an integration test?

Reply:
We missed an opportunity to test that the fields are being mapped properly. But we can easily correct that: in `test_lmsassignments.py`, at line 106, we can test the individual field assignments, to make sure that each value in `lmsx.Assignment` was assigned correctly.